nextflow_schema.json

```diff
@@ -56,6 +49,17 @@
                     "default": "alevin",
```

Comment:
This param should probably also be an enumeration `enum`, so that it can validate that the options are valid, right?

Reply:
Absolutely, still need to have another look over the Schema :)